x-pack/test/security_solution_cypress/cypress/e2e/ai_assistant/conversations.cy.ts

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EXPLAIN_THEN_SUMMARIZE_RULE_DETAILS,
+  RULE_MANAGEMENT_CONTEXT_DESCRIPTION,
+} from '@kbn/security-solution-plugin/public/detections/pages/detection_engine/rules/translations';
+import { EXPLAIN_THEN_SUMMARIZE_SUGGEST_INVESTIGATION_GUIDE_NON_I18N } from '@kbn/security-solution-plugin/public/assistant/content/prompts/user/translations';
+import {
+  assertConnectorSelected,
+  assertNewConversation,
+  closeAssistant,
+  openAssistant,
+  selectConnector,
+  createNewChat,
+  selectConversation,
+  assertMessageSent,
+  typeAndSendMessage,
+  assertErrorResponse,
+  selectRule,
+  assertErrorToastShown,
+  updateConversationTitle,
+  assertSystemPrompt,
+} from '../../tasks/assistant';
+import { deleteConversations } from '../../tasks/api_calls/assistant';
+import {
+  azureConnectorAPIPayload,
+  bedrockConnectorAPIPayload,
+  createAzureConnector,
+  createBedrockConnector,
+} from '../../tasks/api_calls/connectors';
+import { expandFirstAlert } from '../../tasks/alerts';
+import { ALERTS_URL } from '../../urls/navigation';
+import { waitForAlertsToPopulate } from '../../tasks/create_new_rule';
+import { visitRulesManagementTable } from '../../tasks/rules_management';
+import { deleteAlertsAndRules, deleteConnectors } from '../../tasks/api_calls/common';
+import { createRule } from '../../tasks/api_calls/rules';
+import { getExistingRule, getNewRule } from '../../objects/rule';
+import { login } from '../../tasks/login';
+import {
+  CONNECTOR_MISSING_CALLOUT,
+  PROMPT_CONTEXT_BUTTON,
+  USER_PROMPT,
+} from '../../screens/ai_assistant';
+import { visit, visitGetStartedPage } from '../../tasks/navigation';
+
+describe(
+  'AI Assistant Conversations',
+  {
+    tags: ['@ess', '@serverless'],
+    env: {
+      ftrConfig: {
+        productTypes: [
+          { product_line: 'security', product_tier: 'complete' },
```

Comment:
The cloud add-on has an impact on the test?

Reply:
AI Assistant is only available in complete